plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -176,9 +175,7 @@ func IncrementUsage(a admission.Attributes, status *api.ResourceQuotaStatus, cli
 	obj := a.GetObject()
 	// handle max counts for each kind of resource (pods, services, replicationControllers, etc.)
 	if a.GetOperation() == admission.Create {
-		// TODO v1beta1 had camel case, v1beta3 went to all lower, we can remove this line when we deprecate v1beta1
-		resourceNormalized := strings.ToLower(a.GetResource())
```

Comment:
Yes I think this is fine. We do not allow camelCase at many other places (looks like we deleted them when we stopped supporting beta1,2).

@bgrant0607 to confirm.